frontend/src/state/GraphState.jsx

```diff
@@ -427,79 +428,86 @@ export class GraphContainer {
     let displayOrder = 0;
     let rows = {
       displayedRows: {
-        name: "Visible rows", displayOrder: displayOrder++,
+        name: "Visible rows",
```

Comment:
## `name`, `displayOrder` on separate lines
I can change this back if you want; it's kinda arbitrary. ChatGPT liked this better and I know that having them on separate lines abides by code styles, and I'm undecided which I like better.